### PR TITLE
cli-tools: Add nvd

### PR DESCRIPTION
**Pull request by jtojnar**

### CODE REVIEW

**Review by houstdav000 — CHANGES_REQUESTED** [review]
A quick grammatical fix, and it's good.

**Comment by houstdav000** [comment]
Hmm...I think the failing check might be one of those GitHub webapp bugs, given the character looks like it might not be the ASCII quote.

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@
 * [nix-index](https://github.com/bennofs/nix-index) - Quickly locate Nix packages with specific files.
 * [nix-prefetch](https://github.com/msteen/nix-prefetch) - A universal tool for updating source checksums.
 * [nix-tree](https://github.com/utdemir/nix-tree) - Interactively browse the dependency graph of Nix derivations.
+* [nvd](https://gitlab.com/khumba/nvd) - Diff package versions between two store paths; it’s especially useful for comparing NixOS generations on rebuild.
```
Comment (houstdav000) [inline README.md]:
```suggestion
* [nvd](https://gitlab.com/khumba/nvd) - Diff package versions between two store paths; it's especially useful for comparing NixOS generations on rebuild.
```

Comment (houstdav000) [reply README.md]: Lemme see if this fixes it, or if it's just this "suggestion" apply thing

**Review by houstdav000 — APPROVED** [review]
Looks good.

**Comment by jtojnar** [comment]
It is reported here https://github.com/sindresorhus/awesome-lint/issues/104

**Pull request merged**